packages/apps/src/SideBar/index.tsx

```diff
@@ -2,74 +2,50 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import { I18nProps } from '@polkadot/ui-app/types';
+import { I18nProps } from "@polkadot/ui-app/types";
 
-import './SideBar.css';
+import "./SideBar.css";
```

Comment:
We use single quotes. Hence CI failing the checks, i.e. https://travis-ci.org/polkadot-js/apps/jobs/460754993#L571

Reply:
Okay, I will make the requested change, my IDE must have automatically added it.